www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -166,6 +172,11 @@ function (declare, dom, Deferred, ObjectStore, Store, QueryResults, topic,
         reportData.reviewComment = review.author && review.comment
           ? review.comment
           : review.author ? '-' : '';
+
+        reportData.bugPathLength = {
+          length : reportData.bugPathLength,
+          max : maxBugPathLength
```

Comment:
Why do we need the max here?

Reply:
We set a background color for each bug path length. We calculate this background color by using this maximum value (maximum length will be red). For more information see: `generateRedGreenGradientColor` in `util.js`.

Reply:
But do we need this inside all report? Or no better place to put this? Also, this is the max on the displayed page or max in the run? In case the former, I wonder if that is the behavior the users are expecting.